crates/api/src/lib.rs

```diff
@@ -18,13 +18,11 @@ mod trap;
 mod types;
 mod values;
 
-pub mod wasm;
-
 pub use crate::callable::Callable;
 pub use crate::externals::*;
 pub use crate::instance::Instance;
 pub use crate::module::Module;
-pub use crate::r#ref::AnyRef;
+pub use crate::r#ref::{AnyRef, HostInfo, HostRef};
```

Comment:
This was re-added back in to get compilation to work, but it was basically necessary anyway due to the implementation details of `AnyRef` which already exposed these types anyway (and I probably shouldn't have removed them in the first place)

Reply:
Would it make sense to use `#[doc(hidden)]` on them to keep the API docs tidier?

Reply:
It might yeah, but I think it's probably best to leave this as-is for now. These types are reachable through the `AnyRef` API anyway which I think we'll still want to tweak, so I think it's best to leave these as-is until we figure out what we want `AnyRef` to look like